util/zenfs.cc

```diff
@@ -164,7 +168,7 @@ void format_path(std::string &path)
 
 int zenfs_tool_list() {
   Status s;
-  ZonedBlockDevice *zbd = zbd_open(true);
+  ZonedBlockDevice *zbd = zbd_open(true, true);
```

Comment:
@yhr I believe here we should not require exclusiveness
```suggestion
  ZonedBlockDevice *zbd = zbd_open(true, false);
```
I don't see any problems with running `zenfs list` on an already mounted ZenFS.
Moreover, `rocksdb.rocksdb_disable_file_deletions` MTR test case depends on this

Reply:
Yes, you are right. There is no harm in listing the files, as it is only the integrity of the file data that is at risk when mounting non-exclusive.